espnet2/tasks/tts.py

```diff
@@ -381,14 +382,104 @@ def build_model(cls, args: argparse.Namespace) -> ESPnetTTSModel:
         assert check_return_type(model)
         return model
 
+    @classmethod
+    def build_model_from_file(
+        cls,
+        config_file: Optional[Union[Path, str]] = None,
+        model_file: Optional[Union[Path, str]] = None,
+        device: str = "cpu",
+    ):
```

Comment:
It seems this modification is general and can also be applied to all other tasks. Could you somehow make the same modification in the ABSTask?